scripts/composer-merge.sh

```diff
@@ -10,6 +10,17 @@ COMPOSER_BUILD=${COMPOSER_BUILD:-composer.build.json}
 COMPOSER_DEV=${COMPOSER_DEV:-composer.dev.json}
 
 echo "==> Create a build composer file"
-# @note: jq needs to exist on host as the merging of Composer config is running
+# @note: gojq needs to exist on host as the merging of Composer config is running
 # before containers are started.
-cat <<< "$(jq --indent 4 -M -s '.[0] * .[1]' composer.json $COMPOSER_DEV )" > ${COMPOSER_BUILD}
+cat <<< "$(gojq -s --indent 4 -M 'def merge(a;b):
```

Comment:
![mind blown](https://media1.tenor.com/images/2acd46917cbfeca0d71d1fd0899f992f/tenor.gif?itemid=4572252)

Reply:
This is a gnarly query

Reply:
`gojq` doesn't have `'.[0] * .[1]'` * operator. /sigh~

Reply:
gojq doesn't support merging 2 json recursively by using `*` operator.